### PR TITLE
Feature/talent page list

### PR DESCRIPTION
**Pull request by KashyretsIvanna**
fix:link, updated rule
feature: List of jobs
added:pagination, connected to be
my task: As a freelancer, I need to view list of jobs to have opportunity to search a relevant job
check: UI, logic
The first image is what the page looks like when it loads
зображення
зображення

### CODE REVIEW

File: apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  });
+
+  const loadMoreData = useCallback(() => {
+    if (loading) {
+      return;
+    }
+    setLoading(true);
+    fetch(
+      'https://randomuser.me/api/?results=10&inc=name,gender,email,nat,picture&noinfo',
```
Comment (lyudad) [inline apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts]: no links in components, only constants and constants

Comment (KashyretsIvanna) [reply apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts]: fixed

File: apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { User } from '../model';
```
Comment (lyudad) [inline apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts]: Снимок экрана 2022-11-20 в 19 58 52

File: apps/freelance/src/redux/services/talents.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const talentApi = createApi({
+  reducerPath: 'talentApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl:
+      'https://randomuser.me/api/?results=10&inc=name,gender,email,nat,picture&noinfo',
```
Comment (lyudad) [inline apps/freelance/src/redux/services/talents.ts]: I already left comments about the links in the code, its unacceptable!

Comment (KashyretsIvanna) [reply apps/freelance/src/redux/services/talents.ts]: fixed

File: apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts (hunk 2 above)
Comment (KashyretsIvanna) [reply apps/freelance/src/pages/TalentListPage/hooks/getTalents.ts]: fixed

File: apps/freelance/src/pages/TalentListPage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  StyledInformation,
+  StyledName,
+} from './styles';
+
+const TalentListPage = (): ReactElement => {
+  const { Search } = Input;
+  const [search, setSearch] = useState<string>('');
+  const { t } = useTranslation();
+  const [page, setPage] = useState<number>(1);
+  const { data } = useGetFreelancerQuery(page);
```
Comment (lyudad) [inline apps/freelance/src/pages/TalentListPage/index.tsx:24]: its mess state should be near state, hook, near hook, constants near constants

**Review by lyudad — APPROVED** [review]

File: apps/freelance/src/pages/TalentListPage/index.tsx (hunk 4 above)
Comment (KashyretsIvanna) [reply apps/freelance/src/pages/TalentListPage/index.tsx:24]: fixed

**Pull request merged**